hash/sega_beena_cart.xml

```diff
@@ -717,53 +1929,53 @@ license:CC0-1.0
+			<dataarea name="page6" size="0x01e2b00c"> <!-- book pages -->
+				<rom name="0 - 0006.png" size="0x01e2b00c" crc="1dc6ee4e" sha1="a6d96b70759d50b54fbfff91b03e63a7270907d7"/>
 			</dataarea>
-			<dataarea name="page7" size="0x1cff872"> <!-- book pages -->
-				<rom name="0 - 0007.png" size="0x1cff872" crc="1a607e99" sha1="16d3a2f2788b38f451e8cbb44e6a31525586ae5f"/>
+			<dataarea name="page7" size="0x01cff872"> <!-- book pages -->
+				<rom name="0 - 0007.png" size="0x01cff872" crc="1a607e99" sha1="16d3a2f2788b38f451e8cbb44e6a31525586ae5f"/>
 			</dataarea>
-			<dataarea name="page8" size="0x1ad9529"> <!-- book pages -->
-				<rom name="0 - 0008.png" size="0x1ad9529" crc="6701731b" sha1="d2b6718907513239f664e184cfa5d150d6957289"/>
+			<dataarea name="page8" size="0x01ad9529"> <!-- book pages -->
+				<rom name="0 - 0008.png" size="0x01ad9529" crc="6701731b" sha1="d2b6718907513239f664e184cfa5d150d6957289"/>
 			</dataarea>
-			<dataarea name="page9" size="0x1b85a1c"> <!-- book pages -->
-				<rom name="0 - 0009.png" size="0x1b85a1c" crc="5670b958" sha1="84576ba7e1155f3155f65b42af757d41aee80934"/>
+			<dataarea name="page9" size="0x01b85a1c"> <!-- book pages -->
+				<rom name="0 - 0009.png" size="0x01b85a1c" crc="5670b958" sha1="84576ba7e1155f3155f65b42af757d41aee80934"/>
 			</dataarea>
-			<dataarea name="page10" size="0x1c6296e"> <!-- book pages -->
-				<rom name="0 - 0010.png" size="0x1c6296e" crc="66df1325" sha1="49778e96652a113785e3d75b0e6de17d4ebce4ef"/>
+			<dataarea name="page10" size="0x01c6296e"> <!-- book pages -->
+				<rom name="0 - 0010.png" size="0x01c6296e" crc="66df1325" sha1="49778e96652a113785e3d75b0e6de17d4ebce4ef"/>
 			</dataarea>
-			<dataarea name="page11" size="0x1d3f5a5"> <!-- book pages -->
-				<rom name="0 - 0011.png" size="0x1d3f5a5" crc="63fe33c2" sha1="130acb3c9317e21f2fee4e524226ade5df85cec2"/>
+			<dataarea name="page11" size="0x01d3f5a5"> <!-- book pages -->
+				<rom name="0 - 0011.png" size="0x01d3f5a5" crc="63fe33c2" sha1="130acb3c9317e21f2fee4e524226ade5df85cec2"/>
 			</dataarea>
-			<dataarea name="page12" size="0x1c292dc"> <!-- book pages -->
-				<rom name="0 - 0012.png" size="0x1c292dc" crc="c47162c8" sha1="0cb2a72e71082f5ded491dac93562977af6fa984"/>
+			<dataarea name="page12" size="0x01c292dc"> <!-- book pages -->
+				<rom name="0 - 0012.png" size="0x01c292dc" crc="c47162c8" sha1="0cb2a72e71082f5ded491dac93562977af6fa984"/>
 			</dataarea>
 		</part>
 	</software>
 
 	<software name="pokedp" supported="no">
-		<description>Pocket Monsters Diamond &amp; Pearl Pokemon o Sagase! Meiro de Daibouken!</description>
+		<description>Pocket Monsters Diamond &amp; Pearl Pokémon o Sagase! Meiro de Daibouken!</description>
 		<year>2009</year>
 		<publisher>Sega Toys</publisher>
-		<info name="alt_title" value="ポケットモンスターダイヤモンド・パール ポケモンをさがせ！めいろでだいぼうけん！"/>
+		<info name="alt_title" value="ポケットモンスターダイヤモンド・パールポケモンをさがせ！めいろでだいぼうけん！"/>
```

Comment:
Same here – it says “Pocket Monster” in Japanese (not “Monsters”).  Also, there should probably still be some separation between the title and subtitle in Japanese, not just jamming them together with no space.